Blog: v2.5.10 (#465)
files-community/Website: src/routes/blog/posts/announcing-files-v2-5-10/+page.md

---
title: Announcing Files, version 2.5.10
description: Details pane, Git columns, open in VS Code & more
thumbnail: /blog-resources/files2-5-10/HeroImage.jpg
date: 6/27/2023
author: files-community
---

We're excited to announce that version 2.5.10 is now available. If you already have Files, you'll get a notification in the app to update. Otherwise, you can download Files for free from our [download page](/download/).

**TL;DR:** This update adds Git columns to the details layout, an open VS Code button in Git repos, and improvements to the details pane. Continue reading to learn more about these changes.

## What's new in v2.5.10

### Git columns in the details layout

> Opening a Git repo in the details layout will now display additional columns with information about the Git status, last commit, commit hash, and commit author. These columns can be turned on or off by right clicking the column headers.

Reviewer: Changed "turned on and off" for "toggle"

```suggestion
Opening a Git repo in the details layout will now display additional columns with information about the Git status, last commit, commit hash, and commit author. These columns can be toggled by right clicking the column headers.
```
